lib/components/note-preview/index.tsx

```diff
@@ -118,12 +117,16 @@ export const NotePreview: FunctionComponent<Props> = ({
           const taskIndex = Array.prototype.indexOf.call(allTasks, node);
 
           let matchCount = 0;
-          const content = note.content.replace(/[\ue000|\ue001]/g, (match) =>
-            matchCount++ === taskIndex
-              ? match === '\ue000'
-                ? '\ue001'
-                : '\ue000'
-              : match
+
+          const content = note.content.replace(
+            /(- \[x\]|- \[ \])/g,
```

Comment:
note also that we have specific rules that these checkboxes must be the first non-whitespace characters on a line. the rule wasn't implemented with the private use area Unicode characters because they exist as exceptions and should not be found except where there's a checkbox. `- [ ]` is much more readily-entered and we probably want to guard against false positives

Reply:
https://github.com/Automattic/simplenote-electron/pull/2461